src/workloads/query/multiplanner/MultiPlanningReadsALotOfData.yml

```diff
@@ -155,11 +155,11 @@ AutoRun:
   - When:
       mongodb_setup:
         $eq:
-          - standalone
+          - single-replica
+          - single-replica-all-feature-flags
+          - single-replica-sbe
+          - single-replica-classic-query-engine
           - standalone-80-feature-flags
-          - standalone-all-feature-flags
-          - standalone-classic-query-engine
-          - standalone-sbe
           - replica
```

Comment:
There's some inconsistencies in the variants we're running across these workloads. Some have `replica` and `replica-all-feature-flags`, others just have `replica-all-feature-flags`. Assuming that's unintentional can we make them all consistent?